test/foundry/MemberTeller.t.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity ^0.8.7;
+
+import "forge-std/Test.sol";
+import "../mocks/MockMemberToken.sol";
+import "../../contracts/MemberTeller.sol";
+
+contract MemberTellerInternalTest is Test, MemberTeller {
```

Comment:
Aren't these tests duplicated in js? Should we keep one or the other?

Reply:
I deleted most of the js unit tests, the remaining js tests are integration and test state updates of flows across multiple contracts